jina/types/arrays/document.py

```diff
@@ -21,6 +21,9 @@
 from ...helper import typename, cached_property, cache_invalidate
 from ...proto import jina_pb2
 
+import numpy as np
+from math import inf
```

Comment:
math is a system dependency, so it goes with the first group.

We do:

```python
# system
import os
import typing
import math

#external
import numpy as np
import torch

#jina dependencies
import ..types.
```

Reply:
ok